zipline/testing/fixtures.py

```diff
@@ -481,10 +482,26 @@ class WithTradingEnvironment(WithAssetFinder,
     :class:`zipline.finance.trading.TradingEnvironment`
     """
     TRADING_ENV_FUTURE_CHAIN_PREDICATES = None
+    RESOURCE_DIR = join('tests/resources')
 
     @classmethod
     def make_load_function(cls):
-        return None
+        """
+        Load benchmark returns and treasury curves data from a static csv file.
+
+        This ensures that we have data over a consistent date range for the
+        entirety of the zipline tests, and guarantees that the data covers the
+        dates required for tests on both the equity and futures calendars.
+        """
+        benchmark_csv_path = join(cls.RESOURCE_DIR, '^GSPC_benchmark.csv')
```

Comment:
What do you think of moving this behavior to `ensure_benchmark_data`, so it would first check for the repo resource, then the current `~/.zipline` data, then download, at each step checking for the date range?

Then this isn't specific to tests, but covers the tests, and means that in many cases, one can run zipline without needing to download benchmarks.